ftdetect/slim.vim

```diff
@@ -1 +1 @@
-autocmd BufNewFile,BufRead *.slim setf slim
+autocmd BufNewFile,BufRead *.slim setfiletype slim
```

Comment:
In that case `setf` was fine. I was just confused as to why you changed it. Not trolling :)